README.md

```diff
@@ -202,29 +202,8 @@ Please see the [official docs site](https://charmed-kubeflow.io/docs) for comple
 
 ## Setting Custom Images for Kserve controller
 
-KServe controller comes with a set of preconfigured images that are used in Kserve workloads. This is the list of default images used in charm.
+KServe controller comes with a set of preconfigured images that are used in Kserve workloads. The list of default images used in the charm can be found in [default-custom-images.json](https://github.com/canonical/kserve-operators/blob/track/0.10/charms/kserve-controller/src/default-custom-images.json).
```

Comment:
As @orfeas-k nicely mentioned, we could use relative paths here so that we don't tie ourselves with a branch (or have to update)

Reply:
good point, committed